portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,10 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <h1>Vincent's Portfolio</h1>  
+    <h1>Vincent's Portfolio</h1>
+    <div id="links">
+        <a href="index2.html">Gaming</a>
```

Comment:
Perhaps this should be called 'gaming.html' instead of 'index2'?

index2 doesn't really give any indication of what's in it